assemblies/assembly-monitoring-with-amazon-prometheus.adoc

```diff
@@ -0,0 +1,19 @@
+= Monitoring with Amazon Prometheus
+
+You can configure {product} to use Amazon Prometheus for comprehensive logging.
+
+.Prerequisites
```

Comment:
https://redhat-documentation.github.io/modular-docs/#assembly-guidelines

Assembly prerequisites

Prerequisites are conditions that must be satisfied before the user can start following the assembly and are applicable to all the modules in the assembly.

Use the second level heading syntax for the Prerequisites section in the assembly so that it is displayed in the table of contents and is consistent with the Additional resources or Next steps sections in the assembly.


```suggestion
== Prerequisites

```